src/Nethermind/Nethermind.Core/Caching/ClockCache.cs

```diff
@@ -86,14 +88,16 @@ private bool SetSlow(TKey key, TValue val)
 
         _cacheMap[key] = new LruCacheItem(offset, val);
         KeyToOffset[offset] = key;
+        _count++;
```

Comment:
shouldn't all `count` interactions be interlocked?

Reply:
No they are all under lock; but using the field means the ConcurrentDictionary doesn't lock the reads to get a consistent count